Doc deploying failed on master

https://travis-ci.org/JuliaLang/julia/jobs/309184110#L2267

```julia
WARNING: importing deprecated binding Base.base64decode into Documenter.
Documenter: setting up target directory.
Documenter: pushing new documentation to remote: github.com/JuliaLang/julia.git:gh-pages.
ERROR: LoadError: Base.base64decode has been moved to the standard library package Base64.
Restart Julia and then run `using Base64` to load it.
Stacktrace:
 [1] error(::Function, ::String, ::String, ::String, ::String, ::String, ::String) at ./error.jl:44
 [2] (::getfield(Documenter, Symbol("##8#15")){String,String,String,String,String,String,SubString{String}})(::Any) at /home/travis/build/JuliaLang/julia/doc/deps/v0.7/Documenter/src/Documenter.jl:432
 [3] mktempdir(::getfield(Documenter, Symbol("##8#15")){String,String,String,String,String,String,SubString{String}}, ::String) at ./file.jl:392
 [4] cd(::getfield(Documenter, Symbol("##7#14")){String,String,String,String,Void,String,String,SubString{String}}, ::String) at ./file.jl:70
 [5] #deploydocs#4(::String, ::String, ::String, ::String, ::String, ::String, ::String, ::String, ::Void, ::Void, ::typeof(deploydocs)) at /home/travis/build/JuliaLang/julia/doc/deps/v0.7/Documenter/src/Documenter.jl:411
 [6] (::getfield(Documenter, Symbol("#kw##deploydocs")))(::Array{Any,1}, ::typeof(deploydocs)) at ./<missing>:0
 [7] top-level scope at /home/travis/build/JuliaLang/julia/doc/make.jl:184
 [8] include_relative(::Module, ::String) at ./loading.jl:509
 [9] include(::Module, ::String) at ./sysimg.jl:15
 [10] process_options(::Base.JLOptions) at ./client.jl:340
 [11] _start() at ./client.jl:406
in expression starting at /home/travis/build/JuliaLang/julia/doc/make.jl:184
make: *** [deploy] Error 1
make: Leaving directory `/home/travis/build/JuliaLang/julia/doc'
```

--- a/doc/REQUIRE
+++ b/doc/REQUIRE
@@ -1,3 +1,3 @@
-Compat 0.32.0 0.32.0+
+Compat 0.39.0 0.39.0+
 DocStringExtensions 0.4.1 0.4.1+
-Documenter 0.12.0 0.12.0+
+Documenter 0.12.4 0.12.4+

